src/protocols/index.js

```diff
@@ -6,24 +6,51 @@ const EthereumABI = require('./ethereum/abi')
 const EthereumSubgraph = require('./ethereum/subgraph')
```

Comment:
Heavily refactored file, https://github.com/graphprotocol/graph-cli/pull/991/commits/303f0c8941787e1201204ba3f3a05ac9c58f4bec has been made to isolate this change from the rest of the PR. 

Make it much simpler to add new networks. And I spotted a mistake along the way:

```
  getManifestScaffold() {
    switch (this.name) {
      case 'arweave':
        return ArweaveMappingScaffold
      case 'ethereum':
        return EthereumManifestScaffold
      case 'near':
        return NearManifestScaffold
      case 'cosmos':
        return CosmosManifestScaffold
    }
  }
        ```
 
 The `ArweaveMappingScaffold` is wrong, should have been `ArweaveManifestScaffold`

Reply:
I think ideally, we'll need to eventually to make this code more generic and have the different chain hooked through an external piece of code.